heroku.md

```diff
@@ -0,0 +1,182 @@
+app individually.
+
+Only use that shared account to claim ownership or change settings
+that require the owner. Otherwise, use your own Heroku account. Add
+yourself to any projects you want (but at least your own!).
+
+Not in DLAD? Ask someone in DLAD to add you to any projects you are
+working on. The only thing that shared account gives you is billing
+control (aka you aren’t missing anything important).
+
+Need a new app moved into our paid accounts? Set it up with your
+account then transfer ownership to the shared account when it is
+ready for billing. This is how DLAD works and so far it is fine.
+
+Not sure if something is too expensive to use as an add-on? Ask.
+There is no fixed rule, so getting other thoughts on the cost /
+benefit is the best current option.
+
+
+## Pipelines
+
+TL;DR: Use [pipelines](https://devcenter.heroku.com/articles/pipelines).
+They are :mega: :rainbow: :beers: :fireworks:
+
+We have staging sites tied to our Master branches. Merge code into
+master and staging builds will update automatically (after tests
+pass).
+
+Before merging, we have PR builds spin up with staging settings into
+new apps. You can access the link to the apps in the PR itself.
+These are helpful when asking non-developers to sign off on a
+feature before it is merged.
+
+PR builds copy ENV from Staging (dev note: `app.json` does the magic
+on this in project root).
+
+Staging and Production ENV are entirely separate. Changing one does
+NOT change the other. This will make you confused one day.
+
+On deploy or ENV change, the application will restart. It is
+normally a quick blip and doesn’t require downtime announcements.
+
+
+## Production Deploy with Pipelines Cheatsheet
+
+### Tag a release in GitHub with release notes that are useful
+
+The tag is purely informational. It is not currently used to push
+to production using Pipelines. Pushing to prod moves whatever is on
+staging, which is quite likely `master` unless you are doing weird
+things (note: don't do weird things).
+
+### Confirm everything is all good on staging.
+
+Staging probably has `master`. If it works, pushing to prod should
+be :rainbow: unless you forgot you'll need changes to ENV for any new
+features you may be pushing if so, make said changes!
+
+Get stakeholder sign off on the Pipeline staging app if it seems
+appropriate (i.e. only push stuff that has been approved to go live
+and / or is a bug / security fix)… but ideally that has been done
+before merge with PR builds… right? right‽
+
+### Login to Heroku Dashboard
+
+On the app Pipeline page, click "Promote to Production"
+
+Choose the "compare on GitHub option" to confirm the change set is
+indeed what you expect it to be. If it is, go back and click
+"Promote". If it is not, determine why it is not what you expect then
+make decide what to do about that.
+
+### Confirm the app restarts and seems good
+It takes a few seconds to restart. If things are _not_ good, rollback
+in the Dashboard.
```

Comment:
Righto. I'll note that. It's just env and code. Any db you screwed with is your problem. Good call to note that.